Cargo.lock

```diff
@@ -12102,30 +12015,32 @@ dependencies = [
  "serde",
  "sp-api",
  "sp-core",
- "sp-debug-derive 14.0.0 (git+https://github.com/paritytech/polkadot-sdk.git?branch=release-polkadot-v1.7.0)",
+ "sp-debug-derive",
```

Comment:
When we create a PR, we should always check the lockfile. If there is an additional identifier behind the crates dependency name, we have it at least twice in our dependencies, implying some error in our tomls. The best crate to look at is `sp-core` or `sp-std`, as they belong to the innermost substrate crates, and will therefore always be listed twice if there is some error. @DidacSF , @darkfriend77